wwwbase/search.php

```diff
@@ -236,6 +263,21 @@
 }
 
 $results = SearchResult::mapDefinitionArray($definitions);
+
+if(SPOOF_ENABLED && $cuv_spoofed) {
+
+  function replaceSpoofedWord($definition, $cuv) {
+    $pattern = '/<b>(.*)<\/b>(\s.*)/';
```

Comment:
Please add a `/U` modified (ungreedy), other wise the regexp will match everything to the last `</b>`).